UserManagementTeam/src/Profile.vue

```diff
@@ -1,23 +1,39 @@
 <template>
-    <div class="profile-container">
-      <div class="profile-card">
-        <div class="profile-content">
-          <div class="profile-info">
-            <p><strong>User Name:</strong> {{ profile.name }}</p>
-            <p><strong>Email:</strong> {{ profile.email }}</p>
-            <p><strong>Phone:</strong> {{ profile.phone }}</p>
-          </div>
+  <div class="profile-container flex justify-center items-center min-h-screen">
+    <div class="profile-card card w-full max-w-md h-[600px] bg-base-100 custom-card-shadow rounded-3xl p-8">
+      <div class="profile-content card-body mt-10"> 
```

Comment:
profile-container  ,profile-card , profile-content , profile-info not useful you can delete it

Reply:
done